calliope/test/common/test_model/energy_cap_per_storage_cap.yaml

```diff
@@ -30,10 +30,20 @@ techs:
                 energy_cap: 1500000
                 storage_cap: 300000
                 interest_rate: 0.1
+    my_dummy_supply:
+        # Unfortunately, the only supply tech in the region is broken.
+        # (Calliope expects at least one supply technology.)
```

Comment:
This is not a very smart solution to the problem. I still decided to handle the special case in here instead of handling it in the code, because

1. It's a rare special case that people should not run into.
2. I tried handling it in the code, but Calliope is not prepared for this case and breaks in more than one location. First, the set of supply techs does not exist (I'd expect it to be empty, but it simply doesnt exist). Second, after fixing the set, ``split_loc_techs`` breaks with non existing techs. Instead of changing that as well I decided to keep the special case in the test.

Reply:
That's fine for now. If there is ever a practical application for buildings models without supply techs, this could always be revisited.